Include/internal/pycore_object.h

```diff
@@ -91,6 +93,17 @@ static inline void _Py_ClearImmortal(PyObject *op)
         op = NULL; \
     } while (0)
 
+static inline void _Py_SetImmutable(PyObject *op)
+{
+    if(op) {
+        op->ob_region = _Py_IMMUTABLE;
+        // TODO once reference counting across regions is fully working
+        // we no longer need to do this
```

Comment:
```suggestion
        // TODO once reference counting across regions is fully working
        // we no longer need to make all immutable objects immortal
```
I have a habit of coming back to comments like this, and no longer know what I meant by "this".